src/main/java/com/sourcegraph/cody/MyAccountTabPanel.kt

```diff
@@ -79,7 +81,10 @@ class MyAccountTabPanel(val project: Project) : JPanel() {
             button("Upgrade") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/subscription") }
         upgradeButton.component.putClientProperty(DarculaButtonUI.DEFAULT_STYLE_KEY, true)
       }
-      button("Check Usage") { BrowserUtil.browse(ConfigUtil.DOTCOM_URL + "cody/manage") }
+      button("Manage Account") {
+          val username = CodyAuthenticationManager.getInstance(project).getActiveAccount()?.let(CodyAccount::username)
```

Comment:
```
> Task :compileKotlin FAILED
e: file:///Users/tim/Codez/jetbrains/src/main/java/com/sourcegraph/cody/MyAccountTabPanel.kt:85:93 Not enough information to infer type variable R
e: file:///Users/tim/Codez/jetbrains/src/main/java/com/sourcegraph/cody/MyAccountTabPanel.kt:85:110 Unresolved reference: username
```

Reply:
There is no such filed in the `CodyAccount` ;)

```
@Tag("account")
data class CodyAccount(
    @NlsSafe @Attribute("name") override var name: String = "",
    @Attribute("displayName") var displayName: String? = name,
    @Property(style = Property.Style.ATTRIBUTE, surroundWithTag = false)
    override val server: SourcegraphServerPath =
        SourcegraphServerPath.from(ConfigUtil.DOTCOM_URL, ""),
    @Attribute("id") override var id: String = generateId(),
) : ServerAccount() {
```

Reply:
I thought I tried `name` but I mustn't have, because that worked! Thank you!!